docs/operator/README.md

```diff
@@ -312,9 +311,9 @@ Backing up the database should be done on an offline node. A typical approach to
 * Sync the database snapshot to offsite storage
 
 The database snapshot contains no confidential information as long as **only** the
-`/<base path>/chains/alcyone/db` directory is backed up.
+`/<base path>/chains/polymesh_mainnet/db` directory is backed up.
```

Comment:
can you keep the same format as the name of the network? something like `/<base path>/chains/mainnet/db`

Reply:
using just mainnet isn't ideal just in case someone is syncing another substrate network on the same machine.

alcyone was unique enough to use by itself, but for mainnet it would be better to have polymesh in there as well.